frontend/src/Pages/PostPage.tsx

```diff
@@ -1,135 +1,182 @@
-import React, {useEffect, useState} from 'react';
-import styles from './PostPage.module.css';
-import {Link, useLocation, useParams, useSearchParams} from 'react-router-dom';
-import {CommentInfo, PostInfo, PostLinkInfo} from '../Types/PostInfo';
-import PostComponent from '../Components/PostComponent';
+import React, { useEffect, useState } from 'react';
```

Comment:
I think this PR doesn't belong here

Reply:
That's just an example of how formatted file will look like

Reply:
I'd suggest to use gists for this purpose. [Example.](https://gist.github.com/Aivean/c7e2c1a3037ab8b22e16714d2b73c27a/revisions)